---
title: "Venom_no_gaps"
format: html
editor: visual
---

## sebastes venom alignment withut gaps

```{r}
library(ape)
tree_3 <- read.tree("C:/witteLab/Fasta_files/venom_no_gaps.fasta.treefile")
plot(tree_3)
```

I would like to force the outgroup, pg 50 of analysis of phylogenetic evolution in R book. check if the tree is rooted

```{r}
is.rooted(tree_3)
```

not rooted. Next make a root

```{r}
tree_3$tip.label # check the names of the nodes, look for root
```

set root to Acanthopagrus

```{r}
root(tree_3,"Acanthopagrus_latus_2")
```

now lets see if I can view it

```{r}
plot(tree_3)
```

This did not work

```{r}
library(ape)

tree_3_rooted <- root(tree_3,
                      outgroup = "Acanthopagrus_latus_2",
                      resolve.root = TRUE)
```

check

```{r}
plot(tree_3_rooted)
```

```{r}
plot(tree_3_rooted, cex = 0.6) #change label size
```

```{r}
par(mar = c(1, 1, 1, 8))  # increase right margin
plot(tree_3_rooted, cex = 0.6) # change margins
```

change layout

```{r}
plot(tree_3_rooted, type = "cladogram")
```

try circluar

```{r}
plot(tree_3_rooted, type = "fan", cex = 0.6)
```

```{r}
pdf("tree_3_rooted.pdf", width = 12, height = 16)
plot(tree_3_rooted, cex = 0.6)

```

```{r}
par(mar = c(1, 1, 1, 1))
plot(tree_3_rooted,
     type = "fan",
     cex = 0.5,
     no.margin = TRUE)
```

```{r}
pdf("tree_3_rooted.pdf", width = 14, height = 14) #rooted at acanthropagas latus 2

par(mar = c(1, 1, 1, 1))
plot(tree_3_rooted,
     type = "fan",
     cex = 0.5,
     no.margin = TRUE)

#publication ready 
```

```{r}
library(ape)

pdf("tree_aligned.pdf", width = 18, height = 24)

par(mar = c(2, 2, 2, 8))

plot(tree_3_rooted,
     type = "phylogram",
     cex = 0.8,
     label.offset = 0.01)

# Align labels
axisPhylo()


```

root tree around acanthropagus latus 1
```{r}
tree_4_rooted <- root(tree_3,
                      outgroup = "Acanthopagrus_latus_1",
                      resolve.root = TRUE)
pdf("tree_aligned.pdf", width = 18, height = 24)

par(mar = c(2, 2, 2, 8))

plot(tree_3_rooted,
     type = "phylogram",
     cex = 0.8,
     label.offset = 0.01)

# Align labels
axisPhylo()
```

```{r}
# show tree
plot(tree_4_rooted)
```


```{r}
# this code lines up tip labes and makes branches clear
#library(ape)

tree_4 <- ladderize(tree_4_rooted)

plot(tree_4,
     cex = 0.5,
     no.margin = TRUE)
```
now line all the labels up
```{r}
plot(tree_4,
     align.tip.label = TRUE,
     cex = 0.6,
     no.margin = TRUE)
```
fix nodes
```{r}

windows(width = 8, height = 16)

plot(tree_4,
     align.tip.label = TRUE,
     cex = 0.5,
     no.margin = TRUE)

```
save to pdf
```{r}
pdf("tree_4.pdf", width = 14, height = 14) #rooted at acanthropagas latus 1

par(mar = c(1, 1, 1, 1))
plot(tree_4,
      align.tip.label = TRUE,
     type = "phylogram",
     cex = 0.5,
     no.margin = TRUE)

#publication ready 

```

